### PR TITLE
feat(weave): Add a new deployment and service for gorilla parquet

### PR DESCRIPTION
**Pull request by yogeshg**

### CODE REVIEW

**Review by yogeshg — COMMENTED** [review]
No need to for GORILLA_*

File: parquet.tf [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+            name = "ONLY_SERVICE"
+            value = "gorilla-parquet"
+          }
+
+          env {
+            name  = "LOCAL_RESTORE"
+            value = var.local_restore
+          }
+
+          env {
```
Comment (jsbroks) [inline parquet.tf:45]: Do we need to add all these variables to get the parquet service working?

File: parquet.tf [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+resource "kubernetes_service" "parquet" {
+  metadata {
+    name = local.parquet_app_name
+  }
+
+  count = var.parquet_enabled ? 1 : 0
+
+  spec {
+    type = "NodePort"
```
Comment (jsbroks) [inline parquet.tf:213]: Cluster IP would probably be better here. I don't think we need to open this port on every node publicly

File: main.tf [hunk 3]
```diff
@@ -1,6 +1,7 @@
 locals {
   app_name           = "wandb"
   weave_app_name     = "weave"
+  parquet_app_name   = "gorillaparquet"
```
Comment (jsbroks) [inline main.tf:4]:
```suggestion
  parquet_app_name   = "parquet"
```

File: parquet.tf [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
```
Comment (yogeshg) [inline parquet.tf:219]:
```suggestion
    type = "ClusterIP"
```

**Review by jsbroks — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.9.0 :tada:

**Pull request merged**